README.md

```diff
@@ -44,6 +47,89 @@ The content will be arranged in reverse chronological order
 
 ### 2017
 
+#### [April 23th 2017](#april-23th-2017)
+
+##### Module development for wireless systems using Python
+
+This will be hands-on in nature where I will introduce the audience to the
+wireless system, i.e, 4G and alter some parameters in the Python code and show
+the effect.  Pre-requisites needed: A basic understanding of 4G technology; 4G
+technology is the one using which Reliance Jio is offering all the data plans
+right now.
+
+- Speaker: Ashok Govindarajan
+
+##### Introduction to Statecharts 
+
+This talk will be cover the deficiencies of existing State Machine based design,
+and how to overcome them using Statecharts. In the process we will also discuss
+a framework for working with Statecharts, and how to use it to build
+applications.
+
+- Speaker: Vijay Kumar
+- Slides and code: [Dropbox](https://www.dropbox.com/sh/zkujtgo1h68i3nm/AABLRrzG-BPu8TrblVu17rRXa?dl=0)
+---
+
+#### [March 25th 2017](#march-25th-2017)
+
+##### Iteration Protocol in Python
```

Comment:
Ok. I have asked speakers on meetup to provide link to slides.
Apart from that LGTM.
@bravegnu Can you give link for your talk recording?

Thanks